sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileSystems.java

```diff
@@ -434,56 +421,55 @@ static FileSystemRegistrar getRegistrarInternal(String scheme) {
   /********************************** METHODS FOR REGISTRATION **********************************/
 
   /**
-   * Loads available {@link FileSystemRegistrar} services.
+   * Sets the default configuration in workers.
+   *
+   * <p>It will be used in {@link FileSystemRegistrar FileSystemRegistrars} for all schemes.
    */
-  private static void loadFileSystemRegistrars() {
-    SCHEME_TO_REGISTRAR.clear();
+  public static synchronized void setDefaultConfigInWorkers(PipelineOptions options) {
```

Comment:
Also need to synchronize accessors to `SCHEME_TO_FILESYSTEM` -- specifically, `#getFileSystemInternal`.